Travis failure because of Spotbugs

```
[ERROR] Unread field: org.apache.druid.indexing.worker.config.WorkerConfig.intermediaryPartitionCleanupPeriodSec; should this field be static? [org.apache.druid.indexing.worker.config.WorkerConfig] At WorkerConfig.java:[line 49] SS_SHOULD_BE_STATIC
[ERROR] Unread field: org.apache.druid.indexing.worker.config.WorkerConfig.intermediaryPartitionDiscoveryPeriodSec; should this field be static? [org.apache.druid.indexing.worker.config.WorkerConfig] At WorkerConfig.java:[line 46] SS_SHOULD_BE_STATIC
[ERROR] Unread field: org.apache.druid.indexing.worker.config.WorkerConfig.version; should this field be static? [org.apache.druid.indexing.worker.config.WorkerConfig] At WorkerConfig.java:[line 38] SS_SHOULD_BE_STATIC
[INFO] 
```

Probably caused by #8073.

FYI @Fokko

--- a/codestyle/spotbugs-exclude.xml
+++ b/codestyle/spotbugs-exclude.xml
@@ -85,6 +85,7 @@
     <Bug pattern="SF_SWITCH_FALLTHROUGH"/>
     <Bug pattern="SF_SWITCH_NO_DEFAULT"/>
     <Bug pattern="SR_NOT_CHECKED"/>
+    <Bug pattern="SS_SHOULD_BE_STATIC"/>
     <Bug pattern="SWL_SLEEP_WITH_LOCK_HELD"/>
     <Bug pattern="UL_UNRELEASED_LOCK_EXCEPTION_PATH"/>
     <Bug pattern="URF_UNREAD_FIELD"/>

--- a/server/src/main/java/org/apache/druid/server/security/AnonymousAuthenticator.java
+++ b/server/src/main/java/org/apache/druid/server/security/AnonymousAuthenticator.java
@@ -40,8 +40,8 @@ import java.util.Map;
 @JsonTypeName("anonymous")
 public class AnonymousAuthenticator implements Authenticator
 {
-  private static final String DEFAULT_IDENTITY = "defaultUser";
   private final AuthenticationResult anonymousResult;
+  private final String DEFAULT_IDENTITY = "defaultUser";
 
   @JsonCreator
   public AnonymousAuthenticator(

